src/Ellipse.h

```diff
@@ -15,14 +15,14 @@ class Ellipse final : public Model
 
     virtual std::string get_name() const override;
 
+    float center_x = 0.0f;
+    float center_z = 0.0f;
+    float radius_x = 5.0f;
+    float radius_z = 5.0f;
+    i32 segment_count = 40;
```

Comment:
You also need to change the names in the .cpp file, seems like it doesn't compile right now